cql-ast/src/main/scala/troy/cql/ast/Statements.scala

```diff
@@ -33,6 +33,14 @@ final case class SelectStatement(
   allowFiltering: Boolean
 ) extends DataManipulation
 
+final case class UpdateStatement(
+  tableName: TableName,
+  using: Option[Seq[UpdateParam]],
```

Comment:
I think maybe `Seq[UpdateParam]`, no need for `Option`.
If statement didn't have `using` clause, then it will an empty Seq.